README.md

```diff
@@ -153,6 +154,7 @@ https://www.riot-os.org
 [merge-chance-link]: https://merge-chance.info/target?repo=RIOT-OS/RIOT
 [release-badge]: https://img.shields.io/github/release/RIOT-OS/RIOT.svg
 [release-link]: https://github.com/RIOT-OS/RIOT/releases/latest
+[riot-logo]: https://upload.wikimedia.org/wikipedia/commons/b/b8/Logo-RIOT.png
```

Comment:
Why use the wikimedia version? We have our logo [here](https://www.riot-os.org/branding.html) and it is also available as SVG and PNG in `doc/doxygen/src/`?

Reply:
```suggestion
[riot-logo]: doc/doxygen/src/riot-logo.svg
```

Reply:
(add the one from the branding page as `riot-logo-large.png` or something if the one already there is too small)